tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/generate_multishop_combination.feature

```diff
@@ -0,0 +1,184 @@
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    And product "product1" default combination for shop "shop1" should be "product1SWhite"
+    And product "product1" should have no combinations for shops "shop2"
+    And I generate combinations in shop "shop2" for product product1 using following attributes:
+      | Size  | [S,M]              |
+      | Color | [White,Black,Blue] |
+    And product "product1" should have the following combinations for shops "shop2":
+      | combination id | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SWhite | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 0        | true       |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | false      |
+      | product1SBlue  | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhite | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlack | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    And product "product1" default combination for shop "shop2" should be "product1SWhite"
+    Given I delete combination "product1SWhite" from shops "shop1"
+    And product "product1" should have the following combinations for shops "shop1":
+      | combination id | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | true       |
+      | product1SBlue  | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhite | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlack | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    And product "product1" default combination for shop "shop1" should be "product1SBlack"
+    When I generate combinations in shop "shop1" for product product1 using following attributes:
+      | Size  | [S]     |
+      | Color | [White] |
+    Then product "product1" should have the following combinations for shops "shop1":
+      | id reference   | combination id | combination name        | reference | attributes           | impact on price | quantity | is default |
```

Comment:
```suggestion
     | combination id | combination name        | reference | attributes           | impact on price | quantity | is default |
```

Reply:
The combination was not deleted since it is still associated to shop2
So in this case you can rely on `combination id` only which actually asserts that the previously deleted combination (on one shop) is still present and associated again